patternfly/patternfly-elements
# fix(tools): correct dev server not serving lightdom.css files

## What I did

1. Fix dev-server not serving lightdom.css files

Closes #2794 

## Testing Instructions

1. Follow instructions in #2794 ensure `lightdom.css` and `lightdom-shim.css` serve.
2. Launch dev server `npm run serve`

## Notes to Reviewers

1.
